custom/icds_reports/utils/data_pull/governance_dashboard.sql

```diff
@@ -0,0 +1,177 @@
+                     ->  Parallel Append  (cost=0.00..113228.11 rows=141670 width=173)
+                           ->  Parallel Seq Scan on "agg_awc_2019-11-01_5" agg_awc_1  (cost=0.00..112519.76 rows=141670 width=173)
+                                 Filter: ((month = '2019-11-01'::date) AND (aggregation_level = 5) AND (num_launched_awcs = 1))
+                           ->  Parallel Seq Scan on agg_awc  (cost=0.00..0.00 rows=1 width=168)
+                                 Filter: ((month = '2019-11-01'::date) AND (aggregation_level = 5) AND (num_launched_awcs = 1))
+                     ->  Index Scan using awc_location_local_doc_id_idx on awc_location_local awc_location  (cost=0.42..0.62 rows=1 width=240)
+                           Index Cond: (doc_id = agg_awc_1.awc_id)
+                           Filter: ((aggregation_level = 5) AND (agg_awc_1.state_id = state_id) AND (agg_awc_1.district_id = district_id) AND (agg_awc_1.block_id = block_id) AND (agg_awc_1.supervisor_id = supervisor_id))
+         ->  Index Scan using "ix_ucr_icds-cas_static-vhnd_form_28e7fd58_awc_id" on "ucr_icds-cas_static-vhnd_form_28e7fd58" ucr  (cost=0.43..0.84 rows=1 width=49)
+               Index Cond: (awc_id = awc_location.doc_id)
+               Filter: ((vhsnd_date_past_month >= '2019-11-01'::date) AND (vhsnd_date_past_month < '2019-12-01'::date))
+   ->  Seq Scan on icds_months_local months  (cost=0.00..25.00 rows=6 width=4)
+         Filter: (start_date = '2019-11-01'::date)
+(18 rows)
+
+*/
+
+
+
+-- to get children open and valid in oct and Nov in single query. It is needed in single query because they are needed in a single sheet for manipulation
+select
+    state_name,
+    district_name,
+    block_name,
+    supervisor_name,
+    awc_name,
+    SUM(CASE WHEN agg1.age_tranche::INTEGER<=36 THEN agg1.valid_in_month else 0 END) as open_valid_till_month_0_3_Nov,
+    SUM(CASE WHEN agg1.age_tranche::INTEGER between 37 and 72 THEN agg1.valid_in_month else 0 END) as open_valid_till_month_3_6_Nov,
+    SUM(CASE WHEN agg1.age_tranche::INTEGER<=36 THEN agg1.valid_all_registered_in_month else 0 END) as open_register_till_month_0_3_Nov,
+    SUM(CASE WHEN agg1.age_tranche::INTEGER between 37 and 72 THEN agg1.valid_all_registered_in_month else 0 END) as open_register_till_month_3_6_Nov,
+
+    SUM(CASE WHEN agg2.age_tranche::INTEGER<=36 THEN agg2.valid_in_month else 0 END) as open_valid_till_month_0_3_Oct,
+    SUM(CASE WHEN agg2.age_tranche::INTEGER between 37 and 72 THEN agg2.valid_in_month else 0 END) as open_valid_till_month_3_6_Oct,
+    SUM(CASE WHEN agg2.age_tranche::INTEGER<=36 THEN agg2.valid_all_registered_in_month else 0 END) as open_register_till_month_0_3_Oct,
+    SUM(CASE WHEN agg2.age_tranche::INTEGER between 37 and 72 THEN agg2.valid_all_registered_in_month else 0 END) as open_register_till_month_3_6_Oct
+FROM agg_awc_monthly
+left join (select * from agg_child_health where aggregation_level=5 and month='2019-11-01') agg1
```

Comment:
i dont know if it makes a performance difference (althoguh it might becuase it doesnt specify selecting all columns like this one) but i think this would be much easier to read as a normal join rather than joining on a subselect